File: sdks/java/src/main/java/com/puxbay/models/Notification.java
package com.puxbay.models;

import com.google.gson.annotations.SerializedName;

public class Notification {
    private String id;
    private String title;
    private String message;
    
    @SerializedName("is_read")
    private boolean isRead;
    
    @SerializedName("notification_type")
    private String notificationType;
    
    @SerializedName("created_at")
    private String createdAt;

    public String getId() { return id; }
    public void setId(String id) { this.id = id; }
    
    public String getTitle() { return title; }
    public void setTitle(String title) { this.title = title; }
    
    public String getMessage() { return message; }
    public void setMessage(String message) { this.message = message; }
    
    public boolean isRead() { return isRead; }
    public void setRead(boolean read) { isRead = read; }
    
    public String getNotificationType() { return notificationType; }
    public void setNotificationType(String notificationType) { this.notificationType = notificationType; }
    
    public String getCreatedAt() { return createdAt; }
    public void setCreatedAt(String createdAt) { this.createdAt = createdAt; }
}

File: sdks/kotlin/src/main/kotlin/com/puxbay/models/Notification.kt
package com.puxbay.models

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class Notification(
    val id: String,
    val title: String,
    val message: String,
    @SerialName("is_read") val isRead: Boolean,
    @SerialName("notification_type") val notificationType: String,
    @SerialName("created_at") val createdAt: String
)
